fcrepo-audit-triplestore/src/test/java/org/fcrepo/camel/audit/triplestore/integration/AuditSparqlIT.java

```diff
@@ -17,54 +17,68 @@
-import static org.fcrepo.camel.FcrepoHeaders.FCREPO_URI;
-import static org.slf4j.LoggerFactory.getLogger;
-
-import java.util.Map;
-import java.util.HashMap;
-import java.io.IOException;
-
-import org.apache.camel.Produce;
-import org.apache.camel.Exchange;
+import org.apache.camel.CamelContext;
 import org.apache.camel.EndpointInject;
+import org.apache.camel.Exchange;
+import org.apache.camel.Produce;
 import org.apache.camel.ProducerTemplate;
 import org.apache.camel.builder.RouteBuilder;
-import org.apache.camel.builder.xml.XPathBuilder;
-import org.apache.camel.builder.xml.Namespaces;
 import org.apache.camel.component.mock.MockEndpoint;
-import org.apache.camel.test.junit4.CamelTestSupport;
-import org.apache.jena.fuseki.embedded.FusekiEmbeddedServer;
+import org.apache.camel.language.xpath.XPathBuilder;
+import org.apache.camel.spring.javaconfig.CamelConfiguration;
+import org.apache.camel.support.builder.Namespaces;
+import org.apache.jena.fuseki.main.FusekiServer;
 import org.apache.jena.query.Dataset;
 import org.apache.jena.sparql.core.DatasetImpl;
 import org.fcrepo.camel.audit.triplestore.AuditHeaders;
 import org.fcrepo.camel.audit.triplestore.AuditSparqlProcessor;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.support.AnnotationConfigContextLoader;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import static java.lang.Integer.parseInt;
+import static java.util.Arrays.asList;
+import static org.apache.jena.rdf.model.ModelFactory.createDefaultModel;
+import static org.apache.jena.vocabulary.RDF.type;
+import static org.fcrepo.camel.FcrepoHeaders.FCREPO_AGENT;
+import static org.fcrepo.camel.FcrepoHeaders.FCREPO_DATE_TIME;
+import static org.fcrepo.camel.FcrepoHeaders.FCREPO_EVENT_ID;
+import static org.fcrepo.camel.FcrepoHeaders.FCREPO_EVENT_TYPE;
+import static org.fcrepo.camel.FcrepoHeaders.FCREPO_URI;
+import static org.slf4j.LoggerFactory.getLogger;
 
 /**
  * Represents an integration test for interacting with an external triplestore.
  *
  * @author Aaron Coburn
  * @since Nov 8, 2014
  */
-public class AuditSparqlIT extends CamelTestSupport {
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = {AuditSparqlIT.ContextConfig.class}, loader = AnnotationConfigContextLoader.class)
+public class AuditSparqlIT {
```

Comment:
It doesn't seem like the `audit.filter.containers` functionality gets tested anywhere. Seems like it would be helpful to verify that it ignores updates to resources in the routes configured there

Reply:
It's actually there, but buried.  I'll  break out a separate test in RouteTest